server/vitals.js

```diff
@@ -280,6 +281,10 @@ async function handleHeartbeat(req, res) {
             await sendIsTamperedAlert(location, isTamperedFlag, mostRecentSensorVitals.isTampered)
           }
 
+          if (isINSZero) {
```

Comment:
@ahmaddaadaa , it's unclear to me how or whether you tested the case when the INS value was 0. I don't see that test case in your test plan. Were you able to do it?